AirCasting.xcodeproj/project.pbxproj

```diff
@@ -2374,6 +2386,7 @@
 			attributes = {
 				LastSwiftUpdateCheck = 1240;
 				LastUpgradeCheck = 1320;
+				ORGANIZATIONNAME = Lunar;
```

Comment:
Could you tell me what is this change?

Reply:
Yeah, so whenever I was creating a new file I have there my name. After adding this I am back with a proper stamp that says "Lunar ..." . This was the easiest solution